monkey/infection_monkey/post_breach/actions/clear_command_history.py

```diff
@@ -56,9 +57,12 @@ def run(self):
                 if self.command:
                     try:
                         output = subprocess.check_output(  # noqa: DUO116
-                            self.command, stderr=subprocess.STDOUT, shell=True
+                            self.command,
+                            stderr=subprocess.STDOUT,
+                            shell=True,
+                            timeout=LONG_REQUEST_TIMEOUT,
                         ).decode()
                         return output, True
-                    except subprocess.CalledProcessError as e:
+                    except (subprocess.CalledProcessError, subprocess.TimeoutExpired) as err:
                         # Return error output of the command
-                        return e.output.decode(), False
+                        return err.output.decode(), False
```

Comment:
I think there may be a logic error here:

![image](https://user-images.githubusercontent.com/19957806/160902601-745e7ec7-d1a0-49c1-8d03-c9de02ee0283.png)